.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: >-
+      ruby-${{ matrix.ruby }}
```

Comment:
Sets the descriptors seen in the left column:
![Screen Shot 2020-04-17 at 9 19 56 AM](https://user-images.githubusercontent.com/116289/79579121-c9f0ce00-808c-11ea-900d-63ac041a7619.png)